src/dialect/mod.rs

```diff
@@ -114,16 +115,20 @@ pub trait Dialect: Debug + Any {
     fn is_delimited_identifier_start(&self, ch: char) -> bool {
         ch == '"' || ch == '`'
     }
+
```

Comment:
the use of whitespace between functions was inconsistent -- sometimes there was a space and sometimes not. Thus I made it all consistent with my personal preference (put a space between functions0

Reply:
👍 I really wish `rustfmt` would fix this to be consistent.